keras_core/backend/__init__.py

```diff
@@ -37,5 +37,8 @@
 elif backend() == "torch":
     print_msg("Using PyTorch backend.")
     from keras_core.backend.torch import *  # noqa: F403
+elif backend() == "numpy":
+    print_msg("Using NumPy backend.")
```

Comment:
Up to Francois, but I think it might be nice in this situation to add to the end of the print something like.

"The NumPy backend does not support training. It should only be used for evaluation and debugging."

Reply:
for inference, evaluation, and debugging.